CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+We welcome contributions from the community and love getting help from people who want to make this project better.
+
+## How to submit a pull request
+
+All changes must come through a pull request from a forked repository. Please don't commit directly into this repo.
```

Comment:
Only folks with write permission could conceivably commit to the repo, so they wouldn't have expected the second sentence requesting them not to commit directly (since it's not possible usually).

Might be worth highlighting that this is our review mechanism, so even those with write access should issue a pull request for peer review.

We should also mention that PRs are typically against the develop branch and not master, since that is a pristine set of tagged releases, per the git flow model.